.gitignore

```diff
@@ -243,4 +243,5 @@ ModelManifest.xml
 
 # jekyll
 _site
-Gemfile.lock
\ No newline at end of file
+Gemfile.lock
+*.dt
```

Comment:
Is this added by mistake?

Reply:
Yeah, sorry. That is from a time tracking plugin. Feel free to discard.

Reply:
Removed in next commit. No idea Where the GemFile.lock came from so I left it there.